pkgs/development/compilers/gcc/4.9/default.nix

```diff
@@ -424,7 +424,8 @@ stdenv.mkDerivation ({
     platforms =
       stdenv.lib.platforms.linux ++
       stdenv.lib.platforms.freebsd ++
-      stdenv.lib.platforms.illumos;
+      stdenv.lib.platforms.illumos ++
+      stdenv.lib.platforms.darwin;
```

Comment:
IIRC gcc 4.8 is broken on darwin. Perhaps, the same is true for 4.9. Should probably use the ```meta.broken``` attribute to indicate that.